### ISSUE TITLE
Do you have any plans to support the c++20 coroutine？

### ISSUE THREAD

**Issue by nqf** [sewenew/redis-plus-plus#285]

**Comment by sewenew** [comment]
Yes, there’s plan for that, but the priority is low. I’ll add it to the TODO list.

Regards

**Comment by sewenew** [comment]
@nqf Coroutine support for Redis and RedisCluster is done (i.e. `CoRedis`, `CoRedisCluster`). However, coroutine support for sentinel and subscriber is still on the way.

You can take a try, and any feedback or contribution is welcomed!

Regards
